pandas_gbq/tests/test__load.py

```diff
@@ -0,0 +1,26 @@
+
```

Comment:
FYI this file name currently has two underscores

Reply:
Thanks. I'm aware. I'm following the convention that the filename should be `test_` + the filename of the file under test.